crates/iota-sdk-crypto/src/mnemonic.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bip39::Mnemonic;
+
+#[derive(Debug, Clone, Copy)]
+#[non_exhaustive]
+pub enum MnemonicWordCount {
+    Twelve = 12,
+    TwentyFour = 24,
```

Comment:
proposing to rename the variants to something including the numbers, maybe `Wc12, Wc24` or `N12, N24`, but no requirement for approval. 

Reasoning:
(1) more readable
(2) shorter
(3) better aligned

Reply:
I'm going to go ahead and merge, not sure I agree on this one 😬

Reply:
```enum
enum MnemonicLength {
    Words12
    Words24 
}
```
What about this?

Reply:
@Thoralf-M didn't see your message in time. I fully approve on that!!r

Reply:
One more option :trollface: 
```Rust
pub enum MnemonicStrength {
    Bits128,  // 12 words
    Bits256,  // 24 words
}
```